qdata/parser.py

```diff
@@ -71,8 +76,23 @@ def mainprogram(self, *args):
         return self._program
 
     def statement(self, *args):
-        """A catch-all for all OpenQASM statement; this includes register, operator, and gate
-        declarations, include statements, quantum operations to be applied, and barriers.
+        """A catch-all for all OpenQASM statements; this includes register, operator, and gate
+        declarations, ``include`` statements, quantum operations to be applied, and barriers.
+
+        .. code-block:: text
+
+            decl
+            gatedecl goplist "}"
+            gatedecl "}"
+            opdecl goplist "}"
+            opdecl "}"
+            OPAQUE id idlist ";"
+            OPAQUE id "(" ")" idlist ";"
+            OPAQUE id "(" idlist ")" idlist ";"
+            qop
+            IF "(" id "==" nninteger ")" qop
+            BARRIER anylist ";"
+            INCLUDE ESCAPED_STRING ";"
```

Comment:
This last line isn't in the original spec (include seems to be a kind of meta-instruction). Might be confusing?